docs/tutorials/educators/intro.ipynb

```diff
@@ -99,18 +99,7 @@
    "metadata": {
```

Comment:
<p>typo: 'nubmer of repetitions' -&gt; 'number of...'</p><p> </p><p>Adding "cirq." would be good for hrefs on the website, e.g. <code>cirq.Sweep</code>,  <code>cirq.Sampler</code>,  etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='118'/>

Reply:
ack!